src/util/std_code.h

```diff
@@ -659,8 +670,7 @@ inline code_whilet &to_code_while(codet &code)
   return static_cast<code_whilet &>(code);
 }
 
-/*! \brief A `do while' instruction
-*/
+/// \ref codet representation of a `do while' instruction.
```

Comment:
Above you use 'statement', from here you use 'instruction'...

Reply:
Is there a difference? I think it originally said "instruction" everywhere but the background info I found used "statement" more often so I changed it to that in some places.

Reply:
Just as one data point: the C standard exclusively uses the term "instruction" when referring to microprocessor instructions. At source code level, those are statements. Consequently an "instruction" is thus an atom, while a statement may (and does) contain further statements.

In short: I'd go with "statement" everywhere.

Reply:
@tautschnig Changed instruction to statement everywhere (including "goto instruction" to "goto statement" - does that make sense or should that be changed back?).
@allredj Thanks, I just copy-pasted those without paying attention to the quotes. Should all be fixed now.